src/middleware/packages/auth/mixins/auth.js

```diff
@@ -63,29 +73,112 @@ const AuthMixin = {
         // TODO make sure token is deleted client-side
         ctx.meta.webId = 'anon';
         return Promise.reject(new E.UnAuthorizedError(E.ERR_INVALID_TOKEN));
+      } else if (method === 'Capability' || req.$params.capability) {
+        const capabilityUri = token || req.$params.capability;
+        const capability = await this.actions
+          .getValidateCapability({ capabilityUri, username: req.parsedUrl.match(/[^/]+/)[0] })
```

Comment:
I have the feeling that the username will not be correct in configs other than Pod provider. Maybe disable this check in this case ?